cpp/CMakeLists.txt

```diff
@@ -312,6 +312,11 @@ if(BUILD_CUML_CPP_LIBRARY)
     src/dbscan/dbscan.cu
     src/decisiontree/decisiontree.cu
     src/glm/glm.cu
+    src/holtwinters/Aion.cpp
```

Comment:
I believe we should get rid of all references to AION, since they we're not making these libs part of a separate AION package and people using cuML will not know what AION is/was. I think we can split this up and make that a second change - like get the basics in and then refactor next.
 Really want to make sure we're not duplicating existing stuff from cuML too.

Reply:
Agree with @JohnZed . I'll also take a look at the c++ side of things in this PR and provide my feedback by early next week.